modules/options-for-notebook-server-environments.adoc

```diff
@@ -157,13 +157,46 @@ a| * Python 3.9
 * Scikit-learn 1.2
 * SciPy 1.10
 
+.2+| HabanaAI
+| 2023.2 (Recommended)
+a| * Python 3.9
+* Habana 1.11
+* TensorFlow 2.12
+* Boto3 1.26
+* Kafka-Python 2.0
+* Matplotlib 3.6
+* Numpy 1.23
+* Pandas 1.5
+* Scikit-learn 1.2
+* SciPy 1.10
+* Kfp-tekton 1.5
+* PyTorch 2.0
+* Elyra 3.15
+
+| 2023.1
+a| * Python 3.8
+* Habana 1.10
+* TensorFlow 2.12
+* Boto3 1.17
+* Kafka-Python 2.0
+* Matplotlib 3.6
+* Numpy 1.23
+* Pandas 1.5
+* Scikit-learn 1.2
+* SciPy 1.10
+* Kfp-tekton 1.5
+* PyTorch 2.0
+* Elyra 3.15
+
 |===
 
 Deployment size:: Specifies the compute resources available on your notebook server.
 +
 *Container size* controls the number of CPUs, the amount of memory, and the minimum and maximum request capacity of the container.
 +
-*Number of GPUs* specifies the number of graphics processing units attached to the container.
+*Accelerators* Specifies the accelerators available on your notebook server.
```

Comment:
Looks like the 's' at the start of 'Specifies' needs to be lowercase here.

Reply:
Fixed.